lib/src/text-input/TextInput.test.js

```diff
@@ -777,11 +771,12 @@ describe("TextInput component asynchronous autosuggest tests", () => {
       <DxcTextInput label="Autosuggest Countries" value="Denm" onChange={onChange} suggestions={callbackFunc} />
     );
     const input = getByRole("combobox");
-    fireEvent.focus(input);
-    await waitForElementToBeRemoved(() => getByText("Searching..."));
     expect(input.value).toBe("Denm");
+    await userEvent.click(getByText("Autosuggest Countries"));
```

Comment:
Same here, there's no need for an `await`.

Reply:
Done